src/renderer/screens/exchange/Swap2/Form/FormSummary/SectionFees.js

```diff
@@ -1,37 +1,82 @@
 // @flow
-import React, { useMemo } from "react";
+import React, { useMemo, useEffect } from "react";
 import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { getAccountUnit } from "@ledgerhq/live-common/lib/account";
 import { context } from "~/renderer/drawers/Provider";
 import SummaryLabel from "./SummaryLabel";
 import SummaryValue from "./SummaryValue";
 import SummarySection from "./SummarySection";
 import FeesDrawer from "../FeesDrawer";
 import sendAmountByFamily from "~/renderer/generated/SendAmountFields";
+import type { SwapTransactionType } from "~/renderer/screens/exchange/Swap2/utils/shared/useSwapTransaction";
+import { rateSelector } from "~/renderer/actions/swap";
+import FormattedVal from "~/renderer/components/FormattedVal";
+import Text from "~/renderer/components/Text";
 
-const SectionFees = ({ swapTransaction }: { swapTransaction: * }) => {
+const SectionFees = ({ swapTransaction }: { swapTransaction: SwapTransactionType }) => {
   const { t } = useTranslation();
   const { setDrawer } = React.useContext(context);
   const { account, transaction } = swapTransaction;
-  const canEdit = transaction?.networkInfo && sendAmountByFamily[account?.currency?.family];
+  const exchangeRate = useSelector(rateSelector);
+  const fromAccount = swapTransaction.swap.from.account;
+  const fromAccountUnit = fromAccount && getAccountUnit(fromAccount);
```

Comment:
ℹ️ Just a thing we can keep in mind for the refactor step: maybe it would be a good idea to save units when the user selects an account like we already do for parentAccount and currency for example.